src/blog/posts/models.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+from blog.models import PrimaryKey, BlogBase
+import typing as t
+
+# SQLAlchemy models
+
+# In future this will be an ORM sqlalchemy model,
+# For now db objects are simple dict
+Post = dict
+
+
+# Pydantic models
+class PostBase(BlogBase):
+    user_id: t.Optional[int]
+    created_at: t.Optional[datetime.date]
+    title: t.Optional[str]
+    content: t.Optional[str]
+
+
+class PostCreate(PostBase):
+    id: PrimaryKey = None
```

Comment:
When we create new post, while the data reaches the endpoint it doesn't have an id. Id is generated before saving the data to DB (or if we use auto increment primary key it will be automatically generated when writing the data to DB). So PostCreate model should not contain the id

Reply:
Then I simply call pass. Because other properties come from an abstract class.